tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/CircuitBreakerTimeoutTest.java

```diff
@@ -1,6 +1,6 @@
 /*
  *******************************************************************************
- * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ * Copyright (c) 2018-2020 Contributors to the Eclipse Foundation
```

Comment:
It should be 2018, 2020

Reply:
If this is wrong, then it's wrong almost everywhere.

If it needs fixed, we should probably do a mass change over the whole repo.

Reply:
Raised #545 for this.